components/clarinet-cli/src/runner/api_v1.rs

```diff
@@ -656,6 +656,7 @@ fn mine_block(state: &mut OpState, args: MineBlockArgs) -> Result<String, AnyErr
                             _ => DEFAULT_EPOCH,
                         },
                     };
+                    // Any changes in the above section to use epoch 2.4 instead?
```

Comment:
It would make sense.
Although this feature (part of Deno test framework) is rarely used and it will be discontinued (as it is) as part of the Clarinet 2 work.
I suggest we don't update bother updating it

Reply:
Sounds good. Do the remaining changes make sense? Also, I added comments in a few other places, can you please review those too? @hugocaillard